skynet/modules/ttt/openai_api/app.py

```diff
@@ -23,10 +23,11 @@ def initialize():
 
     proc = subprocess.Popen(
         f'{openai_api_server_path} \
-            -m {llama_path} \
-            -b {llama_n_batch} \
-            -c {llama_n_ctx} \
-            -ngl {llama_n_gpu_layers} \
+            --batch-size {llama_n_batch} \
+            --ctx-size {llama_n_ctx} \
+            --flash-attn \
```

Comment:
What effect did you observe here?

Reply:
https://www.hopsworks.ai/dictionary/flash-attention

As a result the available space for kv cache increases and there's some improvement with regards to the inference speed